lib/Doctrine/ORM/Tools/Export/Driver/YamlExporter.php

```diff
@@ -48,11 +48,17 @@ public function exportClassMetadata(ClassMetadataInfo $metadata)
         } else {
             $array['type'] = 'entity';
         }
+        $metadataTable = $metadata->table;
```

Comment:
This too can be simplified with the `??` operator

Reply:
Done.